README.md

```diff
@@ -48,7 +50,7 @@ Browsers Requirement
 Prerequisite
 ---
 
-- Node.js 4.x or up (test up to 6.7.0)
+- Node.js 6.x or up (test up to 6.7.0)
```

Comment:
Yeah, we'll test the 6 version.
Above comment just saying that the "test up to" should be node 7